ddl/ddl.go

```diff
@@ -1184,9 +1524,76 @@ func IterAllDDLJobs(txn kv.Transaction, finishFn func([]*model.Job) (bool, error
 	return IterHistoryDDLJobs(txn, finishFn)
 }
 
+// GetLastHistoryDDLJobsIterator gets latest N history DDL jobs iterator.
+func GetLastHistoryDDLJobsIterator(m *meta.Meta) (meta.LastJobIterator, error) {
```

Comment:
This function feels unnecessary, the calls are all in the same file and have only one line of code

Reply:
it is a reserved function, in the future we may change the implementation, and have this function makes caller no need change the code